### ISSUE TITLE
Perception simulator attempts to set an invalid block transformation

### ISSUE THREAD

**Issue by mkoval** [personalrobotics/block_sorting#12]
```
Redecting objects
[INFO] [block_sorting:run_demo.py:102]:<module>: Redecting objects
/home/mkoval/ros-block-sorting/src/prpy/src/prpy/util.py:775: RuntimeWarning: invalid value encountered in add
  center = (min_corner + max_corner) / 2.
Traceback (most recent call last):
  File "/home/mkoval/ros-block-sorting/src/block_sorting/scripts/run_demo.py", line 105, in <module>
    blocks = perception_simulator.DetectBlocks(robot, table, blocks=[])
  File "/home/mkoval/ros-block-sorting/src/prpy/src/prpy/perception/base.py", line 11, in __call__
    return self.func(instance, robot, *args, **kw_args)
  File "/home/mkoval/ros-block-sorting/src/block_sorting/src/block_sorting/perception_simulator.py", line 80, in DetectBlocks
    block.SetTransform(block_in_world)
openravepy._openravepy_.openravepy_ext.openrave_exception: openrave (Assert): [/tmp/buildd/pr-openrave-0.9.2/include/openrave/geometry.h:448] -> OpenRAVE::geometry::RaveTransform<T> OpenRAVE::geometry::RaveTransform<T>::operator*(const OpenRAVE::geometry::RaveTransform<T>&) const [with T = double], expr: t.rot.lengthsqr4() > 0.99f && t.rot.lengthsqr4() < 1.01f
```

@Shushman @mklingen

**Comment by mklingen** [comment]
weird it goes straight from run_demo.py to util.py...

**Comment by mklingen** [comment]
which prpy branch you on?

**Comment by mkoval** [comment]
`feature/perception_pipeline`, include the changes that just got merged in. Those changes are necessary to get to this point, so I don't know if they're related.

I think this is the root error. It looks like an `INF` or `NAN:
```
/home/mkoval/ros-block-sorting/src/prpy/src/prpy/util.py:775: RuntimeWarning: invalid value encountered in add
  center = (min_corner + max_corner) / 2.
```

**Comment by mklingen** [comment]
can't reproduce...

**Comment by mklingen** [comment]
but it seems the thing is failing to compute the AABB of the table. Has your table model in pr-ordata also changed?

**Comment by jeking04** [comment]
I was able to reproduce and fix this. The problem was that if MoveUntilTouch failed during PlanToEndEffectorOffset then the table would not get reenabled. This caused ComputeEnabledAABB to return nan's. The pose of the block in the yaml file is defined in the relation to the corner of the table, so this caused us to compute an invalid transform for the block.

Fixed in the feature/perception_pipeline branch in my fork of prpy.
